quickwit-indexing/src/split_store/local_split_store.rs

```diff
@@ -24,33 +24,40 @@ use std::{fs, io};
 
 use quickwit_common::split_file;
 use quickwit_directories::BundleDirectory;
-use quickwit_storage::{StorageErrorKind, StorageResult};
+use quickwit_storage::{get_split_streamer, StorageErrorKind, StorageResult};
 use tantivy::directory::MmapDirectory;
 use tantivy::Directory;
 use tracing::{error, warn};
 
 use super::IndexingSplitStoreParams;
 
+pub fn get_tantivy_directory_from_split_bundle(
+    split_file: &Path,
+) -> StorageResult<Box<dyn Directory>> {
+    let mmap_directory = MmapDirectory::open(split_file.parent().ok_or_else(|| {
+        io::Error::new(
+            io::ErrorKind::NotFound,
+            format!("Couldn't find parent for {:?}", &split_file),
+        )
+    })?)?;
+    let split_fileslice = mmap_directory.open_read(Path::new(&split_file))?;
+    Ok(Box::new(BundleDirectory::open_split(split_fileslice)?))
+}
+
 #[derive(Clone, Debug)]
-pub struct BundledSplitFile {
+pub struct SplitFolder {
     pub path: PathBuf,
```

Comment:
That's not from your code, but we should either make path non-pub, or if it is not possible, remove the `new` constructor ?

Reply:
done